skimage/data/detect.py

```diff
@@ -0,0 +1,16 @@
+import os as _os
+from .. import data_dir
+
+
+def frontal_face_cascade_xml():
```

Comment:
@stefanv I made it inside the original function because it takes time to load
and parse that file and it's a little bit faster in cython.
Taking into account that usually people will use this framework for detection only on one image, the loading should also be fast.